content/aips/35A56F18FEFC-AIP-REDEEM-AFEI-FOR-DAI.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Ethereum v2 Reserve Factor - aFEI Holding Update
+status: Proposed
+author: Llama, Matthew Graham & defijesus
+shortDescription: Redeem aFEI for DAI
+discussions: https://governance.aave.com/t/arc-ethereum-v2-reserve-factor-afei-holding-update/9401
+created: 28-09-2022
+---
+
+# Simple Summary
+
+This proposal proposes unwinding the aFEI holding for DAI and depositing the DAI into the Aave v2 Ethereum Pool DAI Reserve.
+
+# Abstract
+
+Due to recent events within the Tribe DAO community, this proposal will over time redeem all aFEI in the Ethereum Reserve Factor for FEI. The FEI holding will then be redeemed for DAI via Tribe DAO’s newly deployed SimpleFeiDaiPSM and deposited into the Aave v2 Ethereum DAI Reserve to earn yield, [1].
+
+This will reduce Aave’s exposure to FEI. 
+
+# Motivation
+
+The FEI reserve is concentrated with Aave DAO holding 94.3% of all aFEI supply, [2]. 
+
+The reserve currently supports 662,518.93 units of aFEI, with 582,449.43 units of debt and 80,069.50 units available for lenders to redeem their collateral from or for Users to borrow, [3,4]. Any User seeking to claim 80,069.50 or less is able to do so now, but this may change as others start redeeming aFEI. 
+
+Limited FEI liquidity remains in the Reserve. Aave DAO is dependent on Borrowers repaying their debt which provides the liquidity needed for Aave to redeem the aFEI position.
+
+Once the aFEI has been redeemed for FEI, it is to be redeemed for DAI via Tribe DAO’s SimpleFeiDaiPSM. There is no redeem fee and all FEI is to be redeemed for DAI via this contract.
+
+The SimpleFeiDaiPSM is supported by Tribe DAO as means of removing FEI from circulation, [4]. The SimpleFeiDaiPSM has around $57,4M of liquidity at the time of writing, [1].
+
+# Specification
+
+The below provides a broad, high level summary of how the aFEI position held within the Reserve Factor will be managed. 
+
+Actors can call a contract from Etherscan which queries both Aave’s aFEI holding and the FEI Reserve, determines the lower of the two values, and redeems the maximum amount of aFEI based upon available liquidity within the Reserve
+All FEI is to be redeemed for DAI via the Tribe DAO’s SimpleFeiDaiPSM
+DAI will be deposited in the Aave v2 Ethereum DAI Reserve
+Aave v2 Ethereum Reserve Factor will receive aDAI
+
+# Test Cases
+
+This proposal has been tested and peer reviewed by Bored Ghost Developing, [5].
+
+Test cases can be found: Here
```

Comment:
Can you link https://github.com/llama-community/aave-redeem-fei/blob/main/src/test/ValidateRedeemFei.sol here